samples/boards/reel_board/env_logger/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _env_logger:
+
+Bluetooth: Environmental Logger
+###############################
+
+Overview
+********
+
+Application serves as digital thermometer, with capability of sending its
```

Comment:
This doesn't adequately describe what the sample does or how to build and run it. The code deals with both temperature and humidity but that's not mentioned in this description. What are the steps to configure, build, and flash the sample? What are the steps to get the Linux app configured, built and run? How do I get the "TICK stack" and how to I configure "Telegraf"? Does it need a serial console to see the output? What's written on the Reel board's display (if anything)?